multi-node/generic/controller-install.sh

```diff
@@ -872,8 +947,21 @@ init_flannel
 systemctl stop update-engine; systemctl mask update-engine
 
 systemctl daemon-reload
+
+# TODO(pb): these options together have not been tested. They shouldn't be mutually exclusive options in the future.
```

Comment:
Agreed. We should probably just set "CNI Config" based on calico=true (and default to flannel if not). Then it's just a matter of testing that rkt+calico work nicely together.